### DISCUSSION TITLE
React View for Marks (For Example Link)

### DISCUSSION THREAD

**Discussion by danielyogel** [ueberdosis/tiptap#2796] [General Questions]
Is it possible to create a React view for the link mark?
Thank you!

**Comment by sereneinserenade** [comment]
No, we can't have NodeView/ReactView for marks. You can have the NodeView for the Node and make that node inline take a look at this. https://tiptap.dev/api/schema#inline

<img width="822" alt="image" src="https://user-images.githubusercontent.com/45892659/178129416-c03f53a9-ad7f-44c3-a134-014b1a0211fb.png">
